src/store/account/getters.ts

```diff
@@ -0,0 +1,14 @@
+import { AccountStateType } from "@/store/account/state";
+import { Account, ValidationFailure } from "@/models";
+import { ScreenState } from "@/types";
+
+const getters = {
+  account: (state: AccountStateType): Account => {
+    return state.account;
+  },
+  isLoggedIn: (state: AccountStateType): boolean => state.isLoggedIn,
+  status: (state: AccountStateType): ScreenState => state.status,
```

Comment:
Consider renaming these variables to reflect that these are for account creation.